[ci][git][travis] Browser tests fail on Linux

see 

```
@theia/example-browser: [chrome #0-4] 6 failing
@theia/example-browser: [chrome #0-4]
@theia/example-browser: [chrome #0-4] 1) terminal UI should open a new terminal and then close it:
@theia/example-browser: [chrome #0-4] unknown error: Element <div class="p-MenuBar-itemLabel">...</div> is not clickable at point (357, 14). Other element would receive the click: <p id="notification-text-3-...-|">Could not find Git on the PATH.</p>
@theia/example-browser: [chrome #0-4] Error: An unknown server-side error occurred while processing the command.
@theia/example-browser: [chrome #0-4]     at elementIdClick("0.5138757508818415-9") - click.js:20:22
@theia/example-browser: [chrome #0-4]
@theia/example-browser: [chrome #0-4] 2) call hierarchy view UI call hierarchy view should toggle-on then toggle-off:
@theia/example-browser: [chrome #0-4] unknown error: Element <div class="p-MenuBar-itemLabel">...</div> is not clickable at point (208, 14). Other element would receive the click: <div class="text">...</div>
@theia/example-browser: [chrome #0-4] Error: An unknown server-side error occurred while processing the command.
@theia/example-browser: [chrome #0-4]     at elementIdClick("0.5138757508818415-10") - click.js:20:22
@theia/example-browser: [chrome #0-4]
@theia/example-browser: [chrome #0-4] 3) extensions view UI extensions view should toggle-on then toggle-off:
@theia/example-browser: [chrome #0-4] unknown error: Element <div class="p-MenuBar-itemLabel">...</div> is not clickable at point (208, 14). Other element would receive the click: <div class="text">...</div>
@theia/example-browser: [chrome #0-4] Error: An unknown server-side error occurred while processing the command.
@theia/example-browser: [chrome #0-4]     at elementIdClick("0.5138757508818415-10") - click.js:20:22
@theia/example-browser: [chrome #0-4]
@theia/example-browser: [chrome #0-4] 4) files view UI files view should toggle-on then toggle-off:
@theia/example-browser: [chrome #0-4] unknown error: Element <div class="p-MenuBar-itemLabel">...</div> is not clickable at point (208, 14). Other element would receive the click: <div class="text">...</div>
@theia/example-browser: [chrome #0-4] Error: An unknown server-side error occurred while processing the command.
@theia/example-browser: [chrome #0-4]     at elementIdClick("0.5138757508818415-10") - click.js:20:22
@theia/example-browser: [chrome #0-4]
@theia/example-browser: [chrome #0-4] 5) git view UI git view should toggle-on then toggle-off:
@theia/example-browser: [chrome #0-4] unknown error: Element <div class="p-MenuBar-itemLabel">...</div> is not clickable at point (208, 14). Other element would receive the click: <div class="text">...</div>
@theia/example-browser: [chrome #0-4] Error: An unknown server-side error occurred while processing the command.
@theia/example-browser: [chrome #0-4]     at elementIdClick("0.5138757508818415-10") - click.js:20:22
@theia/example-browser: [chrome #0-4]
@theia/example-browser: [chrome #0-4] 6) search view UI should start with search view visible:
@theia/example-browser: [chrome #0-4] expected false to be true
@theia/example-browser: [chrome #0-4] AssertionError: expected false to be true
@theia/example-browser: [chrome #0-4]     at Context.<anonymous> (/home/travis/build/theia-ide/theia/examples/browser/lib/test/top-panel/top-panel.ui-spec.js:223:65)
@theia/example-browser: [chrome #0-4]     at new Promise (<anonymous>)
@theia/example-browser: [chrome #0-4]     at new F (/home/travis/build/theia-ide/theia/node_modules/core-js/library/modules/_export.js:36:28)
```

from https://api.travis-ci.org/v3/job/496059660/log.txt

it looks like git is not found on the PATH. `git version 2.15.1` should be available, according to system info from logs.

--- a/.travis.yml
+++ b/.travis.yml
@@ -86,8 +86,6 @@ before_install:
   - export PATH=$HOME/.yarn/bin:$PATH ;
 install: yarn && scripts/check_git_status.sh
 script:
-  # Unset the `protocol.version` in the `global` Git configuration. See: https://github.com/theia-ide/theia/issues/4371
-  - git config --global --unset protocol.version
   - travis_retry yarn test:theia ;
   - travis_retry yarn test:electron ;
   - travis_retry yarn test:browser ;
@@ -109,8 +107,6 @@ jobs:
     env: CXX=c++
     before_script: skip
     script:
-      # Unset the `protocol.version` in the `global` Git configuration.
-      - git config --global --unset protocol.version
       - travis_retry yarn test:theia ;
   - stage: deploy
     if: NOT type IN (cron, pull_request)

--- a/examples/browser/wdio.base.conf.js
+++ b/examples/browser/wdio.base.conf.js
@@ -261,7 +261,7 @@ function makeConfig(headless) {
             try {
                 result = browser.execute("return window.__coverage__;");
             } catch (error) {
-                console.error(`Error retreiving the coverage: ${error}`);
+                console.error(`Error retrieving the coverage: ${error}`);
                 return;
             }
             try {

--- a/packages/git/src/node/dugite-git-watcher.slow-spec.ts
+++ b/packages/git/src/node/dugite-git-watcher.slow-spec.ts
@@ -51,11 +51,6 @@ describe('git-watcher-slow', () => {
         repository = { localUri };
 
         await git!.clone('https://github.com/TypeFox/find-git-exec.git', { localUri });
-
-        // This test has different bindings than other Git tests. We want to check the `watcher` here.
-        // Watching repository changes will initialize Git, so we need to get rid of the env variables here after the clone.
-        delete process.env.LOCAL_GIT_DIRECTORY;
-        delete process.env.GIT_EXEC_PATH;
     });
 
     after(function () {

--- a/packages/git/src/node/init/git-init.ts
+++ b/packages/git/src/node/init/git-init.ts
@@ -53,40 +53,39 @@ export class DefaultGitInit implements GitInit {
 
     async init(): Promise<void> {
         const { env } = process;
-        if (typeof env.LOCAL_GIT_DIRECTORY !== 'undefined' || typeof env.GIT_EXEC_PATH !== 'undefined') {
-            await this.handleExternalNotFound('Cannot use Git from the PATH when the LOCAL_GIT_DIRECTORY or the GIT_EXEC_PATH environment variables are set.');
-        } else {
-            try {
-                const { execPath, path, version } = await findGit();
-                if (!!execPath && !!path && !!version) {
-                    // https://github.com/desktop/dugite/issues/111#issuecomment-323222834
-                    // Instead of the executable path, we need the root directory of Git.
-                    const dir = dirname(dirname(path));
-                    const [execPathOk, pathOk, dirOk] = await Promise.all([pathExists(execPath), pathExists(path), pathExists(dir)]);
-                    if (execPathOk && pathOk && dirOk) {
-                        process.env.LOCAL_GIT_DIRECTORY = dir;
-                        process.env.GIT_EXEC_PATH = execPath;
-                        this.logger.info(`Using Git [${version}] from the PATH. (${path})`);
+        try {
+            const { execPath, path, version } = await findGit();
+            if (!!execPath && !!path && !!version) {
+                // https://github.com/desktop/dugite/issues/111#issuecomment-323222834
+                // Instead of the executable path, we need the root directory of Git.
+                const dir = dirname(dirname(path));
+                const [execPathOk, pathOk, dirOk] = await Promise.all([pathExists(execPath), pathExists(path), pathExists(dir)]);
+                if (execPathOk && pathOk && dirOk) {
+                    if (typeof env.LOCAL_GIT_DIRECTORY !== 'undefined' && env.LOCAL_GIT_DIRECTORY !== dir) {
+                        this.logger.error(`Misconfigured env.LOCAL_GIT_DIRECTORY: ${env.LOCAL_GIT_DIRECTORY}. dir was: ${dir}`);
+                        this.messages.error('The LOCAL_GIT_DIRECTORY env variable was already set to a different value.', { timeout: 0 });
                         return;
                     }
+                    if (typeof env.GIT_EXEC_PATH !== 'undefined' && env.GIT_EXEC_PATH !== execPath) {
+                        this.logger.error(`Misconfigured env.GIT_EXEC_PATH: ${env.GIT_EXEC_PATH}. execPath was: ${execPath}`);
+                        this.messages.error('The GIT_EXEC_PATH env variable was already set to a different value.', { timeout: 0 });
+                        return;
+                    }
+                    process.env.LOCAL_GIT_DIRECTORY = dir;
+                    process.env.GIT_EXEC_PATH = execPath;
+                    this.logger.info(`Using Git [${version}] from the PATH. (${path})`);
+                    return;
                 }
-                await this.handleExternalNotFound();
-            } catch (err) {
-                await this.handleExternalNotFound(err);
             }
+            this.messages.error('Could not find Git on the PATH.', { timeout: 0 });
+        } catch (err) {
+            this.logger.error(err);
+            this.messages.error('An unexpected error occurred when locating the Git executable.', { timeout: 0 });
         }
     }
 
     dispose(): void {
         this.toDispose.dispose();
-    }
-
-    // tslint:disable-next-line:no-any
-    protected async handleExternalNotFound(err?: any): Promise<void> {
-        if (err) {
-            this.logger.error(err);
-        }
-        this.messages.info('Could not find Git on the PATH.');
     }
 
 }

--- a/packages/git/src/node/test/binding-helper.ts
+++ b/packages/git/src/node/test/binding-helper.ts
@@ -46,8 +46,6 @@ export function initializeBindings(): { container: Container, bind: interfaces.B
  * For testing only.
  */
 export async function createGit(bindingOptions: GitBindingOptions = GitBindingOptions.Default): Promise<Git> {
-    delete process.env.LOCAL_GIT_DIRECTORY;
-    delete process.env.GIT_EXEC_PATH;
     const { container, bind } = initializeBindings();
     bindGit(bind, {
         bindManager(binding: interfaces.BindingToSyntax<{}>): interfaces.BindingWhenOnSyntax<{}> {

